cmd/exporter/exporter.go

```diff
@@ -79,6 +79,7 @@ var (
 	apiserverEnabled             = flag.Bool("apiserver", true, "if apiserver is disabled, we collect pod information from kubelet")
 	redfishCredFilePath          = flag.String("redfish-cred-file-path", "", "path to the redfish credential file")
 	exposeEstimatedIdlePower     = flag.Bool("expose-estimated-idle-power", false, "estimated idle power is meaningful only if Kepler is running on bare-metal or when there is only one virtual machine on the node")
+	machineSpecFilePath          = flag.String("machine-profile", "", "path to the machine spec file in json format")
```

Comment:
```suggestion
	machineSpecFilePath          = flag.String("machine-profile", "", "path to the machine spec file in json format")
```
Lets use a single terminology ..`machine-spec` or `machine-profile` I am okay with both.  Since you have already called it `machine-spec` and to avoid having to change it everywhere, lets use `--machine-spec` 

sorry about the confusion 👼